changelog/xxxx.feature.rst

```diff
@@ -0,0 +1,3 @@
+`sunpy.map.GenericMap.rsun_meters` now uses `sunpy.map.GenericMap.rsun_obs`
+as a fallback to calculate the radial distance if RSUN_REF metadata isn't
```

Comment:
"radial distance" is potentially confusing, since a reader may think you mean the Sun-observer distance (which is the `radius` in `HeliographicStonyhurst`).

Reply:
👍 , I also changed the property docstring slightly, and went with "assumed radius of observed emission". Other suggestions welcome.